src/modules/deal-investor/deal-investor.service.get-investors.test.ts

```diff
@@ -0,0 +1,68 @@
+import { AcbsAuthenticationService } from '@ukef/modules/acbs/acbs-authentication.service';
+import { AcbsDealPartyService } from '@ukef/modules/acbs/acbs-deal-party.service';
+import { DealInvestorGenerator } from '@ukef-test/support/generator/deal-investor-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { DealInvestorService } from './deal-investor.service';
+
+jest.mock('@ukef/modules/acbs/acbs-deal-party.service');
+jest.mock('@ukef/modules/acbs/acbs-authentication.service');
+
+describe('DealInvestorService', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const idToken = valueGenerator.string();
+  // TODO: replace with value from default configuration, or remove field portfolioIdentifier completely.
+  const portfolioIdentifier = 'E1';
```

Comment:
Constant please.

Reply:
E1 was changed to use constant on Friday. Resolving.